haxepunk/graphics/BitmapText.hx

```diff
@@ -254,14 +227,9 @@ class BitmapText extends Graphic
 	function parseText():Void
 	{
 		// clear current opcode list
-		while (opCodes.length > 0)
-		{
-			opCodes.pop();
-		}
-		while (_scaleStack.length > 0)
-		{
-			_scaleStack.pop();
-		}
+		HXP.clear(opCodes);
```

Comment:
Maybe we could update HXP.clear to use a heuristic, like `if (array.length > 64) array.splice(0, array.length) else while (array.length > 0) array.pop();`